### PR TITLE
PAN-OS - edit-url-filter

### PR DESCRIPTION
**Pull request by bakatzir**
- Fixed an issue where the arguments *override_allow_list*, *override_block_list* of the command ***panorama-edit-url-filter*** were mishandled.

## Status
- [x] Hold - customer verification

## Related Issues
fixes: https://github.com/demisto/etc/issues/41217

## Minimum version of Cortex XSOAR
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests - added on 8.x. no 10.x in the lab.
- [x] Documentation

### CODE REVIEW

**Comment by bakatzir** [comment]
@yuvalbenshalom feel free to review. verified with the customer, all in the referenced issue.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -2372,26 +2372,36 @@ def prettify_get_url_filter(url_filter: dict):
     if 'override' in url_filter:
         override_category_list = url_filter['override']['member']
 
+    if not isinstance(alert_category_list, list):
+        alert_category_list = [alert_category_list]
```
Comment (yuvalbenshalom) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:2376]: why not arg_to_list()? (and for the others)

**Review by yuvalbenshalom — APPROVED** [review]
looks good, see my comment

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 1 above)
Comment (bakatzir) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:2376]: correct, will fix.

**Comment by bakatzir** [comment]
failed on latest server only, at:
```
[2021-09-26 15:32:55] - [MainThread] - [ERROR] - Failed Tests: 
							 - PCAP Parsing And Indicator Enrichment Test (Mock Disabled)
```
merging.

**Pull request merged**